integration/creategetdelete_test.go

```diff
@@ -124,7 +124,7 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
-		Context("and configuring Flux for gitops", func() {
+		Context("and enabling a gitops repo", func() {
```

Comment:
Nit: what does "_and enabling a gitops repo_" mean? 🤔 
Do we want to say "_enabling a repo for gitops_"? "_enabling gitops on the provided repo_"? Or so?
```suggestion
		Context("and enabling gitops on the configured repo", func() {
```

Reply:
yeah... Now I dislike the word "enable" for this :smile: I went with your suggestion. Thanks!

Reply:
Yes, I don't like it too much either 😅 ... but we said we wouldn't change it so... 🤷‍♂